### PR TITLE
Fetching scrollHeight from document.documentElement does not seem to be affected by highly offset div in tests.

### PR DESCRIPTION
**Pull request by jmalloc**
Hi guys, the test "document height is much larger than win height" was failing for me in Chrome and FF. I modified bonzo.doc() to fetch the max of body.scrollWidth/Height and html.scrollWidth/Height which appears to behave more correctly. Also removed some unused vars, etc.

### CODE REVIEW

File: bonzo.js [hunk 1]
```diff
@@ -546,12 +545,10 @@
   };
 
   bonzo.doc = function () {
-    var w = html.scrollWidth,
-        h = html.scrollHeight,
-        vp = this.viewport();
+    var vp = this.viewport();
```
Comment (ded) [inline bonzo.js:548]: why would you add `var` here?

File: bonzo.js [hunk 2]
```diff
@@ -142,8 +141,8 @@
     isNaN(delta[0]) && (delta[0] = isRel ? 0 : el.offsetLeft);
     isNaN(delta[1]) && (delta[1] = isRel ? 0 : el.offsetTop);
 
-    x !== null && (el.style.left = x - offset.left + delta[0] + 'px');
```
Comment (ded) [inline bonzo.js:145]: these next few lines look the same. was a line break added?

File: bonzo.js [hunk 3]
```diff
@@ -10,7 +10,6 @@
       html = doc.documentElement,
       parentNode = 'parentNode',
       query = null,
-      byTag = 'getElementsByTagName',
```
Comment (ded) [inline bonzo.js:13]: nice catch. wasn't even being used

**Comment by ded** [comment]
cool. can you check out the inline comments.

File: bonzo.js (hunk 2 above)
Comment (jmalloc) [inline bonzo.js:145]: Just changed the string literal 'px' to use the variable named px, saves a few bytes as it was already in use elsewhere.

File: bonzo.js (hunk 1 above)
Comment (jmalloc) [inline bonzo.js:548]: I just removed the use of "w" and "h" variables and rolled into a single call to Math.max (as below).

**Comment by jmalloc** [comment]
Done :) sorry for the barrage of commits, I've got to work out how to get GH to stop adding every push to the pull request.

**Comment by ded** [comment]
excellent!

**Pull request merged**